sopel/db.py

```diff
@@ -42,15 +42,15 @@ def _deserialize(value):
 
 class NickIDs(BASE):
     """
-    NickIDs SQLAlchemy Class
+    Nick IDs table SQLAlchemy class.
```

Comment:
This should be on the same line as the opening quotes `"""Nick IDs table SQLAlchemy class."""`.